colossalai/shardformer/policies/bloom.py

```diff
@@ -1,11 +1,26 @@
+import warnings
+from functools import partial
+from types import MethodType
+from typing import Dict, List, Optional, Tuple, Union
+
+import numpy as np
+import torch
 import torch.nn as nn
+from torch import Tensor
+from torch.nn import CrossEntropyLoss, Module
+from transformers.modeling_outputs import BaseModelOutputWithPastAndCrossAttentions
```

Comment:
import of transformers libraries should be inside function

Reply:
These will be done uniformly.